examples/titles.rb

```diff
@@ -0,0 +1,76 @@
+require "shotstack"
+
+configuration = Shotstack::Configuration.new do |config|
+  config.api_key = {"x-api-key" => ENV["SHOTSTACK_KEY"] }
+  config.host = "api.shotstack.io"
+  config.base_path = "stage"
+
+  config
+end
+
+styles = [
+  "minimal",
+  "blockbuster",
+  "vogue",
+  "sketchy",
+  "skinny",
+]
+
+api_client = Shotstack::ApiClient.new(configuration)
+
+soundtrack = Shotstack::Soundtrack.new(
+  effect: "fadeInOut",
+  src: "https://s3-ap-southeast-2.amazonaws.com/shotstack-assets/music/dreams.mp3")
+
+clips = []
+start = 0
+length = 4
+
+styles.each_with_index do |style, index|
+  options = Shotstack::TitleClipOptions.new(
+    style: style,
+    effect: "zoomIn")
+
+  transition = Shotstack::Transition.new(
+    in: "wipeRight",
+    out: "wipeRight")
+
+  clip = Shotstack::TitleClip.new(
+    type: "title",
+    src: style,
+    in: 0,
+    length: length,
```

Comment:
Parameter got mixed up, it should be `out: length`

Reply:
@jeffski Fixed